### PR TITLE
SOPA DE MACACO, UMA DELICIA

### PR DESCRIPTION
**Pull request by Militaires**
:cl: Basilman
add: Added SOPA DE MACACO, UMA DELICIA
/:cl:
image
you need a full set of monkey limbs and a head
EDIT: SOPA DE MACACO DISGUSTS EVERYONE AROUND YOU EXCLUDING YOU

### CODE REVIEW

**Comment by biggiOS** [comment]
uma delicia +1

**Comment by Schwaglord** [comment]
UMA

File: code/modules/food_and_drinks/food/snacks_soup.dm [hunk 1]
```diff
@@ -225,3 +225,11 @@
 	bonus_reagents = list("nutriment" = 4, "vitamin" = 6)
 	tastes = list("beet" = 1)
 	foodtype = VEGETABLES
+
+/obj/item/reagent_containers/food/snacks/soup/monkey
+	name = "sopa de macaco"
+	desc = "Uma delicia"
+	icon_state = "sopademacaco"
+	bonus_reagents = list("nutriment" = 10, "vitamin" = 30)
```
Comment (Mickyan) [inline code/modules/food_and_drinks/food/snacks_soup.dm]: why is this ten times higher than most food types

Comment (Time-Green) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: Because you’re eating an entire body?

Comment (Mickyan) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: vitamins are for healthy food not stuffing a monkey corpse into a bowl

Comment (Time-Green) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: Well an entire body probably has a few vitamins

Comment (Militaires) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: you need a full monkey to make it

**Comment by MrDoomBringer** [comment]
DELICIAAA

**Comment by Shdorsh** [comment]
Do you intend to add the disgust moodlet to eating it?

File: code/modules/food_and_drinks/food/snacks_soup.dm (hunk 1 above)
Comment (Mickyan) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: You need a full monkey to make a super bite burger and a whole bunch of other stuff and it still has half the vitamins than this

Don't pretend you even tried to balance this

Comment (Militaires) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: ok it has more vitamins because it's delicia

**Comment by Militaires** [comment]
uma delicia

**Comment by epicmemename** [comment]
Uma

**Comment by epicmemename** [comment]
Come to brazil

**Comment by Tlaltecuhtli** [comment]
eating it should give disgust to everyone around

**Comment by Militaires** [comment]
ok ill make it disgust everyone around you but make you feel good

**Comment by 81Denton** [comment]
uma

**Comment by MrDoomBringer** [comment]
@Militaires cant you change the potency of the disgust?

File: code/modules/food_and_drinks/food/snacks_soup.dm [hunk 2]
```diff
@@ -225,3 +225,11 @@
 	bonus_reagents = list("nutriment" = 4, "vitamin" = 6)
 	tastes = list("beet" = 1)
 	foodtype = VEGETABLES
+
+/obj/item/reagent_containers/food/snacks/soup/monkey
+	name = "sopa de macaco"
+	desc = "Uma delicia"
```
[the remark this answers is not in the document]
Comment (MrDoomBringer) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: space-brazilian*

**Comment by Shdorsh** [comment]
@nicbn To be included in the same PR, hopefully.

**Comment by octareenroon91** [comment]
>sopa de gondola
>uma infamia

**Comment by Shdorsh** [comment]
Actually, only raising the disgust var in humies should do the trick, as the stomach organ handles the rest, moodlets included:

`
/obj/item/reagent_containers/food/snacks/soup/monkey/attack(mob/M, mob/user, def_zone)
	if(..())
		for(var/mob/living/carbon/human/H in oview(5,M))
			H.disgust += 5
`

Also, there is a  hippie variant of this already where eating it forces the user to say "UMA DELICIA". If you want the user to be included aswell in the disgust, you can use view() instead of oview().

**Comment by nicbn** [comment]
Please make it so that eating sopa of gondola transforms you into Blanka from street fighter

**Comment by Militaires** [comment]
SOPA DE MACACO DISGUSTS EVERYONE AROUND YOU EXCLUDING YOU

**Comment by Ratvar** [comment]
Kill a monkey to get delayed AOE stunweapon? No thanks

**Comment by Militaires** [comment]
**after testing i discovered that it's impossible to stun someone using sopa de macaco because it fills you up way before the disgust levels reach vomit**

unless MAYBE you're like starving idk

**Comment by Shdorsh** [comment]
For your own use and the community's carnivorous sense of humor, I present to you:

Sopa De Gondola

En behandling :::DDDDD

But I'm not a spriter.

**Comment by Shdorsh** [comment]
Here's the icon file with the added sopa de gondola icon in case you want to add it.

[soupsalad.zip](https://github.com/tgstation/tgstation/files/2300430/soupsalad.zip)

**Comment by Kagus** [comment]
Should this also include the GROSS foodtype so humies with deviant tastes will enjoy it as much as eyeball soup and brain cake?

**Comment by Militaires** [comment]
no because then it will gross out normal people for the sake of snowflakes

**Comment by Kagus** [comment]
Well as it is now, the only people who would directly enjoy eating this are the lizards.

Is that really something you want on your hands?

**Comment by Militaires** [comment]
anyone enjoys eating this nigga

File: code/modules/food_and_drinks/food/snacks_soup.dm [hunk 3]
```diff
@@ -225,3 +225,16 @@
...
+	name = "sopa de macaco"
+	desc = "A space-Brazilian delicacy. Uma delicia!"
+	icon_state = "sopademacaco"
+	bonus_reagents = list("nutriment" = 5, "vitamin" = 10)
+	tastes = list("delicia" = 1)
+	foodtype = MEAT
+
+/obj/item/reagent_containers/food/snacks/soup/monkey/attack(mob/M, mob/user, def_zone)
+	if(..())
+		for(var/mob/living/carbon/human/H in oview(M))
```
Comment (optimumtact) [inline code/modules/food_and_drinks/food/snacks_soup.dm]: add a cooldown of some description, add some kind of feedback message and consider the quality of your changes before making them in future

Comment (Militaires) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: a cooldown to what bro

Comment (Militaires) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: if you mean adding a cooldown to the disgust being added which means you cant eat 5 sopa de macaco bowls and gross someone out severely you need to posterize yourself nigga you dont know how this works....

**Comment by optimumtact** [comment]
fine if you wanna play like that

**Comment by Militaires** [comment]
absolutely destroyed

**Comment by octareenroon91** [comment]
oh god what a relief

**Comment by 81Denton** [comment]
outplayed

**Pull request closed without merging**